integration-tests/clipper_admin_tests.py

```diff
@@ -343,6 +344,55 @@ def predict_func(inputs):
             })
         self.assertEqual(len(containers), 1)
 
+    def test_test_predict_function(self):
+        def predict_func(xs):
+            return [sum(x) for x in xs]
+
+        self.clipper_conn.register_application(
+            name="hello-world",
+            input_type="doubles",
+            default_output="-1.0",
+            slo_micros=100000)
+
+        deploy_python_closure(
+            self.clipper_conn,
+            name="sum-model",
+            version=1,
+            input_type="doubles",
+            func=predict_func)
+        self.clipper_conn.link_model_to_app(
+            app_name="hello-world", model_name="sum-model")
+
+        headers = {"Content-type": "application/json"}
+        test_input = list(np.random.random(10))
+        pred = requests.post(
```

Comment:
Set the address the way we set it here: https://github.com/ucbrise/clipper/blob/f9a19d142ce3a0c70aab3386fdd9b055e05b9599/integration-tests/clipper_admin_tests.py#L438-L440